frontend/src/component/executiveDashboard/hooks/useFilteredTrends.test.ts

```diff
@@ -0,0 +1,173 @@
+import { renderHook } from '@testing-library/react-hooks';
+import { useFilteredTrends } from './useFilteredTrends';
+
+const mockProjectFlagTrends = [
```

Comment:
are all the fields relevant from the perspective of this tests? Or is it the minimum set of that to show what the tests does without any noise?

Reply:
Almost, except date. I opted for this instead of type-gymnastics like "Omit<...> & Required<..>"